RULES/libsupcxx/libsupcxx.go

```diff
@@ -1,78 +1,98 @@
 package libsupcxx
```

Comment:
Let's call this package `crossrules` or something similar.

Reply:
crosscc? The Cross* rules are basically a replacement for cc.* rules

Reply:
fine